ni_measurementlink_service/_drivers/__init__.py

```diff
@@ -0,0 +1,130 @@
+"""Types and submodules for interfacing with driver APIs."""
+from __future__ import annotations
+
+import contextlib
+from typing import (
+    ContextManager,
+    Generic,
+    Protocol,
+    Type,
+    TypeVar,
+    cast,
+    runtime_checkable,
+)
+
+import grpc
+
+TSessionInitializationBehavior = TypeVar("TSessionInitializationBehavior", bound=int)
+TSessionInitializationBehavior_co = TypeVar(
+    "TSessionInitializationBehavior_co", bound=int, covariant=True
+)
+
+
+class SessionInitializationBehaviorEnumType(Protocol, Generic[TSessionInitializationBehavior_co]):
+    """Protocol describing a driver API's SessionInitializationBehavior IntEnum type.
+
+    Each driver API has its own version of this IntEnum, which provides the same
+    enum values.
+    """
+
+    # This protocol describes the provided enum values and nothing else.
+    #
+    # Python 3.11 doesn't have a good way to describe read-only class variables
+    # or protocols that inherit from built-in types such as int.
+
+    @property
+    def AUTO(  # noqa: N802 - function name should be lowercase
+        self,
+    ) -> TSessionInitializationBehavior_co:
+        """Automatically initialize a new session or attach to an existing one."""
+        ...
```

Comment:
Is this the same as using `pass`? I've never seen this syntax before and it is hard to come up with a google search to figure out what it means.